File: challenges/codeabbey/021/john2104.py
m,n = raw_input().split()

count = [0]*int(n)
values = raw_input().split()

for i in values:
	count[int(i)-1] = count[int(i)-1]+1

result = ''

for i in count:
	result = result +'%d '%i
print result

File: challenges/codeabbey/002/john2104.ml
let nlines = read_line ();;
let values = read_line ();;
let intlist = List.map int_of_string(Str.split (Str.regexp " ") values);;
let sum = List.fold_left ( + ) 0;;
let result = sum intlist;;
Printf.printf "%d\n" result;;
